sql/core/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveSessionCatalog.scala

```diff
@@ -203,12 +203,13 @@ class ResolveSessionCatalog(val catalogManager: CatalogManager)
     case DescribeRelation(ResolvedV1TableOrViewIdentifier(ident), partitionSpec, isExtended, _) =>
       DescribeTableCommand(ident.asTableIdentifier, partitionSpec, isExtended)
 
-    case DescribeColumn(ResolvedViewIdentifier(ident), column: UnresolvedAttribute, isExtended) =>
+    case DescribeColumn(
+         ResolvedViewIdentifier(ident), column: UnresolvedAttribute, isExtended, _) =>
       // For views, the column will not be resolved by `ResolveReferences` because
       // `ResolvedView` stores only the identifier.
       DescribeColumnCommand(ident.asTableIdentifier, column.nameParts, isExtended)
```

Comment:
shall we pass the output to the v1 command as well?

Reply:
> shall we pass the output to the v1 command as well?

You mean unify V2 and v1 DESC command's output? I think v2's `DESC TABLE` need to do this too.

Reply:
yea, you can fix it here as well.

Reply:
> yea, you can fix it here as well.

Done